src/main/scala/com/amazon/deequ/suggestions/ConstraintSuggestionRunner.scala

```diff
@@ -70,13 +70,17 @@ class ConstraintSuggestionRunner {
       cacheInputs: Boolean,
       fileOutputOptions: ConstraintSuggestionFileOutputOptions,
       metricsRepositoryOptions: ConstraintSuggestionMetricsRepositoryOptions,
-      kllParameters: Option[KLLParameters])
+      kllWrapper: List[Any])
     : ConstraintSuggestionResult = {
 
     // get testset related data from wrapper
     val testsetRatio: Option[Double] = testsetWrapper(0).asInstanceOf[Option[Double]]
     val testsetSplitRandomSeed: Option[Long] = testsetWrapper(1).asInstanceOf[Option[Long]]
 
+    val kllParameters: Option[KLLParameters] = kllWrapper(0).asInstanceOf[Option[KLLParameters]]
+    val predefinedColumnDataTypes: Option[Map[String, String]] =
```

Comment:
What exactly does the kllWrapper and the predefined datatypes have to do with each other?

Reply:
The kllWrapper, which is similar to testsetWrapper listed above, are simply workarounds to pass the argcount requirements when building the whole package. Otherwise I could delete these wrappers and pass the parameters themselves.